### PR TITLE
adds air alarm to morgue on Box Station

### LINKED ISSUE #3681: Medical Morgue on Box Station lacks an air alarm
Issue reported from  Round ID: 27221 (BeeStation Sage - Roleplay)

Reporting client version: 513.1527

image

No air alarm

### PR DESCRIPTION
**Pull request by bloons3**
## About The Pull Request
fixes https://github.com/BeeStation/BeeStation-Hornet/issues/3681

## Why It's Good For The Game
Air alarm in room good

image

## Changelog
:cl:
add: Added an air alarm to the Morgue on Box Station
add: Added vent scrubbers and vent pumps to autopsy rooms
/:cl:

### CODE REVIEW

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -45557,6 +45557,12 @@
 	},
 /turf/open/space/basic,
 /area/space)
+"cIG" = (
+/obj/machinery/airalarm{
+	pixel_y = 23
```
Comment (park66665) [inline _maps/map_files/BoxStation/BoxStation.dmm]:
```suggestion
	pixel_y = 24
```

Comment (bloons3) [reply _maps/map_files/BoxStation/BoxStation.dmm]: Derp, off by one

**Comment by bloons3** [comment]
### 
image


image

**Review by park66665 — APPROVED** [review]
All clear. This PR is good to go.

**Pull request merged**